frontend/src/metabase/search/containers/SearchApp.styled.tsx

```diff
@@ -3,46 +3,36 @@ import {
   breakpointMinMedium,
   breakpointMinSmall,
 } from "metabase/styled-components/theme";
+import { Flex } from "metabase/ui";
 
-export const SearchRoot = styled.div`
-  margin: 0 0.5rem;
+const SEARCH_BODY_WIDTH = "90rem";
+const SEARCH_SIDEBAR_WIDTH = "15rem";
+
+export const SearchMain = styled(Flex)`
+  width: min(calc(${SEARCH_BODY_WIDTH} + ${SEARCH_SIDEBAR_WIDTH}), 100%);
 
   ${breakpointMinSmall} {
-    margin: 0 1rem;
+    padding: 2rem;
   }
 
   ${breakpointMinMedium} {
-    margin: 0 4rem;
+    padding: 2rem 3rem;
   }
 `;
 
-export const SearchHeader = styled.div`
-  display: flex;
-  padding: 1rem 0;
-
-  ${breakpointMinSmall} {
-    padding: 2rem 0;
+export const SearchBody = styled(Flex)`
+  ${breakpointMinMedium} {
+    flex-direction: row-reverse;
```

Comment:
`row-reverse` so that the filters are stacked on top at mobile breakpoints, then on the right side at larger breakpoints